### DISCUSSION TITLE
Where can I find  React projects to contribute?

### DISCUSSION THREAD

**Discussion by Mudassirkhan2** [EddieHubCommunity/support#7293] [General]
Where can I find React js projects?

**Comment by Panquesito7** [comment]
Hey there! 👋

Here are a few resources to find React.js projects
easier, as well as a few React.js projects:

1. GitHub Explore page (you can search by language, JS in this case): https://github.com/explore
2. LinkFree, an awesome project to have all your links together: https://github.com/EddieHubCommunity/LinkFree
3. A great article with amazing 10+ React.js open-source projects: https://themeselection.com/open-source-react-projects/

You can also search for more resources on the web, you'll find a lot of great projects.
I hope that helped. Let us know if you need any more help. 🙂

**Comment by geeky-sambhav** [comment]
> Where can I find React js projects?

For react, 
Rocket chat, zulip, Scorelabs, and some projects are there in Linux foundation as well to contribute...
I hope that helped. Let me know if you need any more help. 🙂

**Comment by Burnsedia** [comment]
The Inifinit Wish Board is a MERN Stack APP

**Comment by EmmaDawsonDev** [comment]
I have some next apps on my organisation Accessible For All and hopefully more coming soon

**Comment by MitAbhay** [comment]
There are ton of places where you can find ReactJS projects @Mudassirkhan2, there is no such specific place we can talk about. You can search it on Google, GitHub, open-source programs like GSOC, LFX, Hacktoberfest and many more.
Hope it helps you.

**Comment by NiazMorshed2007** [comment]
Hi there!
It's good to know you're trying to contribute in open source.
You can eventually find many react projects to contribute here on github.

- You can give a shot to our [Linkfree](https://github.com/EddieHubCommunity/LinkFree) repository
- The Github Explore page where you can filter and find repos
- 🎯Pro tip: Use the [advanced search](github.com/search/advanced) . Advanced search allows you to search more powerfully

Here is a video about advanced searching made by Eddie 👉 https://www.youtube.com/watch?v=L1X5f2ZxN0E

Hope that helps.
